files/files.go

```diff
@@ -77,5 +83,41 @@ func PrepareWikiRoot() error {
 	paths.categoriesJSON = filepath.Join(cfg.WikiDir, "categories.json")
 	paths.interwikiJSON = FileInRoot("interwiki.json")
 
+	// Are we initializing the wiki for the first time?
+	if isFirstInit {
+		err := firstTimeInit()
+		if err != nil {
+			return err
```

Comment:
Though you may want to just log the error without taking down the program because those assets aren't critical at all.